cmd/guacone/cmd/vulnerability.go

```diff
@@ -329,23 +329,40 @@ func searchDependencyPackages(ctx context.Context, gqlclient graphql.Client, top
 	return path, tableRows, nil
 }
 
-func queryVulnsViaVulnNodeNeighbors(ctx context.Context, gqlclient graphql.Client, topPkgResponse *model.PackagesResponse, vulnerabilityNodeID string, edgeType model.Edge, depth, pathsToReturn int) ([]string, error) {
+func queryVulnsViaVulnNodeNeighbors(ctx context.Context, gqlclient graphql.Client, topPkgResponse *model.PackagesResponse, vulnerabilitiesResponses []model.VulnerabilitiesVulnerabilitiesVulnerability, edgeType model.Edge, depth, pathsToReturn int) ([]string, error) {
+	type vulnNeighbor struct {
+		node model.NeighborsNeighborsNode
+		id   string
+	}
+
 	var path []string
-	vulnNodeNeighborResponse, err := model.Neighbors(ctx, gqlclient, vulnerabilityNodeID, []model.Edge{edgeType})
-	if err != nil {
-		return nil, fmt.Errorf("error querying neighbor for vulnerability: %w", err)
+	vulnNodeNeighborResponses := []vulnNeighbor{}
```

Comment:
nit: this should just be `var vulnNodeNeighborResponses []vulnNeighbor` to avoid an unecessary allocation of a 0 length slice.

Reply:
Done, Thanks!